--- src/Makefile.am~	2008-09-09 17:21:15.000000000 +0200
+++ src/Makefile.am	2009-11-15 12:23:56.000000000 +0100
@@ -18,6 +18,4 @@
 	kmfl_load_keyboard.c\
 	kmfl_messages.c
 
-libkmfl_la_LDFLAGS = -lkmflcomp
-
-libkmfl_la_LIBADD = 
+libkmfl_la_LIBADD = -lkmflcomp 
--- src/Makefile.in~	2009-02-04 22:45:32.000000000 +0100
+++ src/Makefile.in	2009-11-15 12:24:19.000000000 +0100
@@ -206,8 +206,8 @@
 	kmfl_load_keyboard.c\
 	kmfl_messages.c
 
-libkmfl_la_LDFLAGS = -lkmflcomp
-libkmfl_la_LIBADD = 
+libkmfl_la_LDFLAGS = 
+libkmfl_la_LIBADD = -lkmflcomp 
 all: all-am
 
 .SUFFIXES:
